content/en/apps/guides/integrations/rapidpro.md

```diff
@@ -83,7 +83,7 @@ For RapidPro to communicate with the CHT, you need to create a [User]({{< ref "a
 
 ### Globals
 
-[Globals](https://blog.textit.in/global-variables) are shared values that can be referenced in flows, as well as broadcasts and campaigns, within your account referenced by `@globals.value_name`. They allow you to create a value once and use it repeatedly without having to reenter the value. Likewise, globals make updating a shared value much easier. Rather than manually changing a value everywhere it's used in your account, simply update the value found in your `Globals` page.
+[Globals](https://help.nyaruka.com/en/articles/3561580-global-variables) are shared values that can be referenced in flows, as well as broadcasts and campaigns, within your account referenced by `@globals.value_name`. They allow you to create a value once and use it repeatedly without having to reenter the value. Likewise, globals make updating a shared value much easier. Rather than manually changing a value everywhere it's used in your account, simply update the value found in your `Globals` page.
```

Comment:
I'm not 100% comfortable with linking to some random third party's post. I think we should drop the link and just include any info/screenshots/etc here directly.

To make this PR simple I propose we just drop the link, and raise a new issue to add anything we're missing.

Reply:
@garethbowen we are linking to [flows](https://help.nyaruka.com/en/articles/3120713-introduction-to-flows), [campaigns](https://help.nyaruka.com/en/articles/2217334-introduction-to-campaigns) and [triggers](https://help.nyaruka.com/en/articles/1911111-introduction-to-triggers) on this page as well, are you proposing we remove all?

Reply:
Oof. Ok. Ignore me. I'm still not 100% comfortable with it but clearly it's a bigger issue than I hoped.